app/src/main/java/zapsolutions/zap/util/Wallet.java

```diff
@@ -676,6 +678,10 @@ public void run() {
                     mPendingForceClosedChannelsList = pendingChannelsResponse.getPendingForceClosingChannelsList();
                     mPendingWaitingCloseChannelsList = pendingChannelsResponse.getWaitingCloseChannelsList();
 
+                    /* Update balance to include limbo channels. The limbo balance does not account for open pending channels.
```

Comment:
Thanks for the review. 
@michaelWuensch  
Do you think we should update the balance only in `fetchBalanceFromLND` and remove it here?

Reply:
Well, it does not harm if we have it here, but actually we don't need it there, so for the sake of cleaner code I would remove it.

Reply:
I've moved it and tested it again. 👍